tests/integrate_test/iiss/decentralized/test_preps_divide_node_address.py

```diff
@@ -0,0 +1,243 @@
+        prev_block_votes = [(x.address, True) for x in self._accounts[1:PREP_MAIN_PREPS]]
+        block, tx_results, _, _, main_prep_as_dict = self.debug_make_and_req_block(tx_list=[],
+                                                                                   prev_block_generator=prev_block_generator,
+                                                                                   prev_block_validators=None,
+                                                                                   prev_block_votes=prev_block_votes,
+                                                                                   block=None)
+        self.assertEqual(tx_results[0].status, True)
+        self.assertEqual(main_prep_as_dict, None)
+
+        self._write_precommit_state(block)
+
+        # 2 after change node_address
+        dummy_node2: 'Address' = create_address()
+        prev_block_generator = dummy_node1_2
+
+        # set prep 2
+        tx: dict = self.create_set_prep_tx(from_=account,
+                                           set_data={"nodeAddress": str(dummy_node2)})
+
+        prev_block_votes = [(x.address, True) for x in self._accounts[1:PREP_MAIN_PREPS]]
+        block, tx_results, _, _, main_prep_as_dict = self.debug_make_and_req_block(tx_list=[tx],
+                                                                                   prev_block_generator=prev_block_generator,
+                                                                                   prev_block_validators=None,
+                                                                                   prev_block_votes=prev_block_votes,
+                                                                                   block=None)
+        self.assertEqual(tx_results[0].status, True)
+        self.assertEqual(main_prep_as_dict["preps"][0]["id"], dummy_node2)
+        self._write_precommit_state(block)
+
+    def test_prep_set_node_address_check_votes(self):
+        self.set_revision(Revision.DIVIDE_NODE_ADDRESS.value)
+
+        self.distribute_icx(accounts=self._accounts[:PREP_MAIN_PREPS],
+                            init_balance=1 * ICX_IN_LOOP)
+
+        # 0 start change node_key
+        account: 'EOAAccount' = self._accounts[1]
+        dummy_node1: 'Address' = create_address()
+        dummy_node1_1: 'Address' = create_address()
+        dummy_node1_2: 'Address' = create_address()
+
+        # set prep 1
+        tx_list: list = []
+
+        tx: dict = self.create_set_prep_tx(from_=account,
+                                           set_data={"nodeAddress": str(dummy_node1)})
+        tx_list.append(tx)
+        tx: dict = self.create_set_prep_tx(from_=account,
+                                           set_data={"nodeAddress": str(dummy_node1_1)})
+        tx_list.append(tx)
+        tx: dict = self.create_set_prep_tx(from_=account,
+                                           set_data={"nodeAddress": str(dummy_node1_2)})
+        tx_list.append(tx)
```

Comment:
Why is "set_prep" called 3 times?

Reply:
the purpose is to check if it works when I called SetPRep many times in a block.